test/e2e/template/helmfile/snapshot_test.go

```diff
@@ -172,6 +182,15 @@ func TestHelmfileTemplateWithBuildCommand(t *testing.T) {
 
 			gotStr := string(got)
 			gotStr = strings.ReplaceAll(gotStr, fmt.Sprintf("chart=%s", wd), "chart=$WD")
+
+			// Replace go-getter path with $GoGetterPath
+			if config.GoGetterMode.Enabled {
+				switch config.GoGetterMode.Protocol {
+				case "git":
+					gotStr = chartGitFullPathRegex.ReplaceAllString(gotStr, `chart=$$GoGetterPath`)
```

Comment:
@yxxhero Can we just extract this out of the two conditional blocks, and remove GoGetterMode altogether?
This replace-all operation seems to have no effect on output that doesn't involve any go-getter charts. Then, executing this unconditionally shouldn't break any non-go-getter tests, right?

Reply:
@mumoshu I got.